daemon/cmd/datapath.go

```diff
@@ -485,16 +492,14 @@ func setupIPSec() (int, uint8, error) {
 	return authKeySize, spi, nil
 }
 
-func setupIPCacheVTEPMapping() error {
-	encryptKey := uint8(0)                           // no encrypt support
-	vtepID := uint32(identity.ReservedIdentityWorld) //network policy identity for VTEP
+func setupVTEPMapping() error {
```

Comment:
Why VTEP and no longer IPCache? In what regard is this an improvement? This info should ideally be documented in the commit description, so that we can understand the reason for this change when looking at the history.